docs/core/examples/java/Assets.java

```diff
@@ -7,6 +7,7 @@
 class Assets {
   public static void main(String[] args) throws Exception {
     Client client = new Client();
+    Map<String, Object> tagUpdate;
```

Comment:
@chrisgarvin This declaration should go inline, on line 77 below. Otherwise, it will be unclear what `tagUpdate` is in the snippet itself.